membership-attribute-service/test/acceptance/PaymentUpdateControllerAcceptanceTest.scala

```diff
@@ -393,13 +391,12 @@ class PaymentUpdateControllerAcceptanceTest extends AcceptanceTest {
       httpResponse.getStatus shouldEqual 200
 
       identityMockClientAndServer.verify(identityRequest)
-      subscriptionServiceMock.current[SubscriptionPlan.Contributor](contact)(any) was called
-      catalogServiceMock.unsafeCatalog was called
```

Comment:
I don't know why it expected this to be called for updating payment details.  I think it was probably called as a side effect.